components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/listener/AbstractWorkflowListener.java

```diff
@@ -775,4 +775,28 @@ public int getOrderId() {
         }
         return workflowListener.getOrder();
     }
+
+        /**
+         * Trigger before retrieving a workflow request.
+         *
+         * @param requestId ID of the workflow request to retrieve
+         * @throws WorkflowException
+         */
+        @Override
+        public void doPreGetWorkflowRequest(String requestId) throws WorkflowException {
+
+        }
+
+        /**
+         * Trigger after retrieving a workflow request.
+         *
+         * @param requestId ID of the workflow request that was retrieved
+         * @param workflowRequest Workflow request object returned by the original operation
+         * @throws WorkflowException
+         */
+        @Override
+        public void doPostGetWorkflowRequest(String requestId, WorkflowRequest workflowRequest) throws WorkflowException {
+                
+        }
```

Comment:
We will not need these methods after addressing this comment https://github.com/wso2/carbon-identity-framework/pull/6894#discussion_r2173661538